src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateField.tsx

```diff
@@ -57,51 +60,65 @@ export const PhrasalTemplateField = ({
 
   return (
     <StyledFlexTopCenter sx={{ gap: 0.8, width: '100%' }}>
-      <RenderedField
-        name={name}
-        responseOptions={responseOptions}
-        type={type}
-        {...otherProps}
-        sx={{ '.MuiFormHelperText-root': { mb: 0.4 } }}
-      />
-      <StyledFlexTopCenter
-        onBlur={handleBlurMenu}
-        onMouseEnter={handleMouseEnterMenu}
-        onMouseLeave={handleHoverLeaveMenu}
-        ref={menuRef}
-        sx={{ gap: 0.8 }}
-      >
-        {showExpandedMenu ? (
-          <>
-            <IconButton color="default" disabled={!canRemove} onClick={onRemove}>
-              <Svg
-                aria-label={t('phrasalTemplateItem.btnRemoveField')}
-                fill="currentColor"
-                id="trash"
-              />
-            </IconButton>
-
-            {/* TODO: M2-7183 — Draggable re-ordering of phrase fields */}
-            <IconButton color="default" disabled>
-              <Svg
-                aria-label={t('phrasalTemplateItem.btnOrderField')}
-                fill="currentColor"
-                id="drag"
-              />
-            </IconButton>
-          </>
-        ) : (
-          <IconButton color="default" onFocus={handleFocusMenu}>
-            <Svg
-              aria-label={t('phrasalTemplateItem.btnShowFieldControls')}
-              fill="currentColor"
-              id="dots"
-              height={18}
-              width={18}
+      <Draggable index={index} draggableId={itemId}>
+        {(draggableProvided) => (
+          <StyledFlexTopCenter
+            width="100%"
```

Comment:
When the `<StyledFlexTopCenter>` was moved here, we lost the `gap`, and now the child components are missing the 8px spacing:

![CleanShot 2024-08-22 at 18 16 35@2x](https://github.com/user-attachments/assets/88cc6f6b-23d5-4848-b107-b01b699e3c91)

vs. Figma:
![CleanShot 2024-08-22 at 18 10 35](https://github.com/user-attachments/assets/393aa1d6-469d-4fb6-b148-d4e351f34cdf)

Easy fix, though! Also the `width="100%"` is not needed, since `<div>`s already stretch the full width of parent block elements (also providing explicit width doesn't play well with negative margins anyway, which I use in my next suggestion 👇🏻 )
```suggestion
            sx={{ gap: 0.8 }}
```